prisma/schema.prisma

```diff
@@ -220,7 +220,7 @@ model ApplicationProcessing {
   appMailed         Boolean           @default(false) @map("app_mailed")
   createdAt         DateTime          @default(now()) @map("created_at") @db.Timestamptz(6)
   updatedAt         DateTime          @default(now()) @map("updated_at") @db.Timestamptz(6)
-  application       Application?      @relation("application_processingToapplications")
+  applications      Application?      @relation("application_processingToapplications")
```

Comment:
```suggestion
  application       Application?      @relation("application_processingToapplications")
```
Since it would return at most one application that is not in an array